miasm/analysis/depgraph.py

```diff
@@ -482,7 +482,7 @@ def visit_inner(self, expr, *args, **kwargs):
             else:
                 self.nofollow.add(expr)
                 return None
-        elif expr.is_function_call():
+        elif expr.is_op() and expr.is_function_call():
```

Comment:
Maybe we can mimic the python code:
bind `is_function_code` to each miasm expr so that we don't have to test the nature of expr
(is_function_code points is to have a shortcut to test this. so having a first test to do before is a bit counter productive ?)

Reply:
Requires a modification to the Rust code, implemented in https://github.com/serpilliere/miasm-rs/pull/76